dev/bots/analyze_sample_code.dart

```diff
@@ -455,7 +456,7 @@ class SampleChecker {
       }
     }
     if (!silent)
-      print('Found ${sections.length} sample code sections.');
+      print('Found ${sections.length} snippet code blocks and ${samples.length} sample code sections.');
```

Comment:
We have snippets, samples, and dartpads - maybe also add which of these numbers includes the dartpad ones?

Reply:
Good idea.  Done.